niftypet/nipet/img/pipe.py

```diff
@@ -35,6 +35,7 @@ def mmrchain(
 
     itr=4,          # number of OSEM iterations
     fwhm=0.,        # Gaussian Smoothing FWHM
+    fwhm_rm=0.,     # Resolution Modelling
```

Comment:
```suggestion
    fwhm_rm=2.5,     # Resolution Modelling
```

might be better to be 2.5mm by default.

Reply:
The default I would prefer to be raw with no added modelling

Reply:
but, when switching on rm, then this can be as default, no problem

Reply:
ok leaving as default=0 for now. "switching on RM" would be the same as specifying a non-zero value. We could also change this later to accepting an entire kernel rather than just a Gaussian FWHM.